ui/imports/shared/views/NetworkSelector.qml

```diff
@@ -1,129 +1,120 @@
-                                    StatusListItem {
-                                        id: item
-                                        implicitWidth: 126
-                                        title: modelData.chainName
-                                        subTitle: ""
-                                        image.source: Style.png("networks/" + modelData.chainName.toLowerCase())
-                                        image.width: 32
-                                        image.height: 32
-                                        leftPadding: 5
-                                        rightPadding: 5
-                                        color: "transparent"
-                                        border.color: Style.current.primary
-                                        border.width: root.selectedNetwork.chainId === modelData.chainId ? 1 : 0
-                                        onClicked: {
-                                            root.selectedNetwork = modelData
-                                            root.networkChanged(modelData.chainId)
-                                        }
-                                    }
-                                }
-                            }
-                        }
-                    }
+        Rectangle {
+            id: simple
+            radius: 13
+            color: Theme.palette.indirectColor1
+            NetworksSimpleRoutingView {
+                id: networksSimpleRoutingPage
+                anchors.top: parent.top
+                anchors.left: parent.left
+                anchors.margins: Style.current.padding
+                selectedNetwork: root.selectedNetwork
+                suggestedRoutes: root.suggestedRoutes
+                amountToSend: root.amountToSend
+                onNetworkChanged:  {
+                    root.selectedNetwork = network
+                    root.networkChanged(network.chainId)
                 }
             }
         }
+
+        Rectangle {
+            id: advanced
+            radius: 13
+            color: Theme.palette.indirectColor1
+            NetworksAdvancedCustomRoutingView {
+                id: advancedNetworkRoutingPage
+                anchors.top: parent.top
+                anchors.left: parent.left
+                anchors.margins: Style.current.padding
+                store: root.store
+                assets: root.assets
+                selectedNetwork: root.selectedNetwork
+                selectedAccount: root.selectedAccount
+                amountToSend: root.amountToSend
+                requiredGasInEth: root.requiredGasInEth
+                selectedAsset: root.selectedAsset
+                suggestedRoutes: root.suggestedRoutes
+                onReCalculateSuggestedRoute: root.reCalculateSuggestedRoute(disabledNetworks)
+            }
+        }
+
+        Rectangle {
+            id: custom
+            radius: 13
+            color: Theme.palette.indirectColor1
```

Comment:
I am thinking the only advantage would be to not have to change radius and color in case design changes? so for that I will create a property that updated all them to the correct values?

Cause I am thinking what would I name this Rect lol